specification/slm/_types/SnapshotLifecycle.ts

```diff
@@ -74,10 +74,10 @@ export class Statistics {
 }
 
 export class Policy {
```

Comment:
Is this another case of `Policy`/`PolicyRead`? If so can we split into two and use `OverloadOf`?

Reply:
The issue was reported for a read operation, and I checked the code: parsing and serialization are symmetrical and have the same tests for those optional properties.